set-body/src/Components/SetBodyStep.tsx

```diff
@@ -0,0 +1,68 @@
+import {Expression} from './Expression';
+import {ActionGroup, Button, Form } from '@patternfly/react-core';
+import {useState} from 'react';
+
+type SetBodyStepParams = {
+  constant?: string;
+  simple?: string;
+  jq?: string;
+};
+
+export const SetBodyStep = (props: any) => {
+  let initSyntax;
+  let initExpression;
+  if (props.stepParams?.jq) {
+    initSyntax = 'jq';
+    initExpression = props.stepParams?.jq || '';
+  } else if (props.stepParams?.constant) {
+    initSyntax = 'constant';
+    initExpression = props.stepParams?.constant || '';
+  } else {
+    initSyntax = 'simple';
+    initExpression = props.stepParams?.simple || '';
+  }
+
+
+  const [stepParams, setStepParams] = useState<SetBodyStepParams>({
+        constant: props.stepParams?.constant,
+        simple: props.stepParams?.simple,
+        jq: props.stepParams?.jq,
+    });
+
+  function updateKaoto() {
+    props.notifyKaoto('Set Body step updated');
+    props.updateStepParams(stepParams);
+  }
+
+  function updateStepParams(constant?: string, simple?: string, jq?: string) {
+    const newStepParams: SetBodyStepParams = {
+      constant: constant,
```

Comment:
Here you can just do `constant` instead of `constant: constant` through ES6 object initializer property definitions!: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions

Reply:
ooooh OK it's even shorter! I'll follow up in #14 as this PR was already auto-merged :laughing: